asa/ja-jp/Content/Topics/Adv_Server_Access/docs/asa-overview.htm

```diff
@@ -223,135 +140,108 @@
-                                                <!-- Back to Top button --><a href="#" class="sf-back-to-top"><span class="arrow"></span>Top</a>
-                                            </div>
+                                            <div class="body-main">                                                 <!-- Main content body -->                                                 <div role="main" id="mc-main-content">
+                                                    <h1>アドバンストサーバーアクセス</h1>
+                                                    <p><span class="mc-variable okta-variables.ProductName variable">Okta</span> <span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>は、クラウドおよびオンプレミスインフラストラクチャのためのZero Trust（ゼロトラスト）IDとアクセス管理を提供します。<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>は、<span class="mc-variable okta-variables.ProductName variable">Okta</span>を信頼できる情報源として、LinuxサーバーおよびWindowsサーバーへのSSHおよびRDPアクセスを管理するためのアカウントを調整します。 </p>
+                                                    <p><span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>は安全な特権アクセスをユーザーに拡張し、サーバーアカウントのライフサイクル管理を自動化して、資格情報を管理する必要性を排除します。</p>
+                                                    <div class="tiles">
+                                                        <div>
+                                                            <p class="tile-title"><a href="start-here.htm">はじめに</a>                                                             </p>
+                                                            <p class="tile-subtitle"><span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>のチームをセットアップし、<span class="mc-variable okta-variables.ProductName variable">Okta</span>からユーザーとグループをプッシュします。</p>
+                                                        </div>
+                                                        <div>
+                                                            <p class="tile-title">サーバーエージェント</p>

```

Comment:
Do we miss <a> tag here?

Reply:
![image](https://user-images.githubusercontent.com/22689859/186285288-663479a0-028e-4029-9ed4-5b26ba1cecf3.png)

Reply:
Looks like we missed those links in the English source at some point
https://github.com/okta/okta-help/blob/a2e65f842806ff51d777a7d1029dc5f2c3cd4443/asa/en-us/Content/Topics/Adv_Server_Access/docs/asa-overview.htm#L318